installer/common/tiller.sh

```diff
@@ -17,40 +17,40 @@
 
 # Returns the namespace where tiller is running or where it should be installed, by default we install it in kube-system
 export_tiller_namespace() {
-    echo "Checking for tiller"
-	tillerNamespace=$(kubectl get pods -l name=tiller --all-namespaces --ignore-not-found=true --no-headers | awk -v i=1 -v j=1 'FNR == i {print $j}')
-	if [ -z "$tillerNamespace" ]; then 
-		return 1
-	fi 
-    TILLER_NAMESPACE="$tillerNamespace"
-    export TILLER_NAMESPACE
-	return 0
+  echo "Checking for tiller"
+  tillerNamespace=$(kubectl get pods -l name=tiller --all-namespaces --ignore-not-found=true --no-headers | awk -v i=1 -v j=1 'FNR == i {print $j}')
```

Comment:
Please revert this if changing indentation is the only change here.

Reply:
fixed